exporter/datadogexporter/internal/translator/metrics_translator.go

```diff
@@ -140,6 +140,41 @@ func (t *Translator) mapNumberMonotonicMetrics(name string, slice pdata.NumberDa
 	return ms
 }
 
+func getBounds(p pdata.HistogramDataPoint, idx int) (lowerBound float64, upperBound float64) {
+	lowerBound = math.Inf(-1)
+	upperBound = math.Inf(1)
+	if idx > 0 {
+		lowerBound = p.ExplicitBounds()[idx-1]
+	}
+	if idx < len(p.ExplicitBounds()) {
+		upperBound = p.ExplicitBounds()[idx]
+	}
+	return
+}
+
+func (t *Translator) getLegacyBuckets(name string, p pdata.HistogramDataPoint, delta bool, tags []string) []datadog.Metric {
+	// We have a single metric, 'bucket', which is tagged with the bucket id. See:
```

Comment:
This is tagged by the bounds, not the bucket id, isn't it?

Reply:
I updated the comment (it was a copy-paste)